api/src/pcapi/routes/backoffice_v3/filters.py

```diff
@@ -382,11 +376,48 @@ def format_offer_validation_operator(operator: offers_models.OfferValidationRule
             return operator.value
 
 
+def format_offer_validation_sub_rule(sub_rule: offers_models.OfferValidationSubRule) -> str:
+    try:
+        sub_rule_field = offers_models.OfferValidationSubRuleField(
+            {
+                "model": sub_rule.model,
+                "attribute": sub_rule.attribute,
+            }
+        )
+        sub_rule_text = format_offer_validation_sub_rule_field(sub_rule_field)
+    except ValueError:
+        sub_rule_text = f"{sub_rule.model.value if sub_rule.model else ''}.{sub_rule.attribute.value}"
+
+    sub_rule_text += f" {format_offer_validation_operator(sub_rule.operator)}"
+    return sub_rule_text
+
+
+def get_comparated_format_function(
+    sub_rule: offers_models.OfferValidationSubRule,
+    offerer_dict: dict,
+) -> typing.Callable[[typing.Any], typing.Any]:
+    try:
+        if (
+            sub_rule.attribute == offers_models.OfferValidationAttribute.ID
+            and sub_rule.model == offers_models.OfferValidationModel.OFFERER
+        ):
+            return lambda offerer_id: offerer_dict[offerer_id]
+        if sub_rule.attribute == offers_models.OfferValidationAttribute.CATEGORY:
+            return lambda category_id: categories.ALL_CATEGORIES_DICT[category_id].pro_label
+        if sub_rule.attribute == offers_models.OfferValidationAttribute.SUBCATEGORY_ID:
+            return lambda subcategory_id: subcategories_v2.ALL_SUBCATEGORIES_DICT[subcategory_id].pro_label
+        if sub_rule.attribute == offers_models.OfferValidationAttribute.SHOW_SUB_TYPE:
+            return lambda show_sub_type_code: SHOW_SUB_TYPES_LABEL_BY_CODE[int(show_sub_type_code)]
+    except ValueError:
+        logger.error("Unhandled object in the formatter of the offer validation rules list page")
```

Comment:
C'est bien de donner des billes au shérif :)
```suggestion
    except ValueError as err:
        logger.error("Unhandled object in the formatter of the offer validation rules list page", extra={"sub_rule_id": sub_rule.id, "error": err})
```